config/routes.rb

```diff
@@ -15,6 +15,8 @@
     get "nonprofits"
   end
 
+  resources :student_applications, only: [:new, :create], path: "apply/students"
```

Comment:
Yeah, it was just to make it look a little nicer! Because they hit that link after going from apply/students